doc/lua_api.txt

```diff
@@ -453,6 +459,10 @@ the global `minetest.registered_*` tables.
 * `minetest.register_craftitem(name, item definition)`
     * added to `minetest.registered_items[name]`
 
+* `minetest.unregister_item(name)`
+    * erased from minetest.registered_items[] and from item associated table
```

Comment:
Hmm please say me what would be the correct sentence, really I am not english-native